tests/rpc/rpc_01-system_test.go

```diff
@@ -111,8 +111,8 @@ func TestSystemRPC(t *testing.T) { //nolint:tparallel
 
 		expectedResponse := modules.SystemPeersResponse{
 			// Assert they all have the same best block number and hash
-			{Roles: 4, PeerID: ""},
-			{Roles: 4, PeerID: ""},
+			common.PeerInfo{Roles: 4, PeerID: ""},
+			common.PeerInfo{Roles: 4, PeerID: ""},
```

Comment:
why ?

Reply:
linter complaining

Reply:
tried running `make lint`, did not throw error. But I see the point of making it explicit that it is common.PeerInfo.

How about 
```
		expectedResponse := modules.SystemPeersResponse(
			[]common.PeerInfo{
				// Assert they all have the same best block number and hash
				{Roles: 4, PeerID: ""},
				{Roles: 4, PeerID: ""},
			},
		)
```

Reply:
just ran `make lint` after install go 1.19. did not complain.

Reply:
Oh well nothing complains anymore, I guess mystery unresolved, conversation resolved!